source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,60 @@
+RoboRIO2 Imaging
+================
+
+.. note:: The imaging instructions for the NI roboRIO 1.0 are :ref:`here <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`.
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the microSD card -- with the roboRIO :ref:`using the imaging tool <docs/zero-to-robot/step-3/imaging-your-roborio:Imaging your roboRIO>`, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.  It is recommended to use a SDHC or SDXC capacity card, though all types are expected to function correctly.
+
+SD (secure digital) cards are the oldest, least used and limited to 2GB of storage.  SDHC (high capacity) cards can store up to 32 GB of data, while SDXC (extended capacity) cards can store up to 2 terabytes (2000 GB).
```

Comment:
```suggestion
```

I suggest removing these sentences unless there is a specific purpose we are explaining the difference between SD, SDHC, and SDXC cards are. I don't really think it fits in the article, perhaps added into the previous paragraph where it can then be edited to the following:
```
The NI roboRIO 2.0 controller supports SDHC (High Capacity) and SDXC (Extended Capacity) microSD cards with a recommended minimum capacity of 2GB. 
```

Reply:
See above.

Reply:
I agree with Dalton on this sentence. I don't think it adds anything that's useful for our purposes. It will still be in the manufacturers documentation.